.github/workflows/build-and-push-container-image.yml

```diff
@@ -29,15 +37,15 @@ jobs:
 
       - name: Get Tags
         run: |
-            if [ ${{ steps.isVersioningCommit.outputs.isVersioningCommit }} ]; then
+            if ${{ steps.isVersioningCommit.outputs.IS_VERSIONING_COMMIT }}; then
                 tagVersion=${{ steps.version.outputs.VERSION }}
-                tags="octopusdeploy/nfs-server:$tagVersion,octopusdeploy/nfs-server:latest";
+                tags="octopusdeploy/nfs-server:$tagVersion,docker.packages.octopushq.com/octopusdeploy/nfs-server:$tagVersion,octopusdeploy/nfs-server:latest,docker.packages.octopushq.com/octopusdeploy/nfs-server:latest";
             else
                 tagVersion=$(git describe --tags)
-                tags="octopusdeploy/nfs-server:$tagVersion,docker.packages.octopushq.com/octopusdeploy/nfs-server:$tagVersion,octopusdeploy/nfs-server:latest,docker.packages.octopushq.com/octopusdeploy/nfs-server:latest";
+                tags="docker.packages.octopushq.com/octopusdeploy/nfs-server:$tagVersion";
```

Comment:
No we're only publishing the git describe versioned image to artifactory when it's not a release build.

When it's a release build we publish it as the release version and `latest` to docker hub and artifactory.